translations/en-us.yaml

```diff
@@ -4420,29 +4420,21 @@ clusterNew:
       placeholder: e.g. 172.16.0.0/16
       required: Virtual Cloud Network CIDR is required
       error: Virtual Cloud CIDR format error
+    vcnCompartment:
+      label: VCN Compartment OCID
+      placeholder: the OCID of the VCN Compartment
     vcnId:
-      help: The OCID of an existing VCN to be used by the cluster
-      label: Virtual Cloud Network OCID
-      placeholder: VCN OCID
-      required: Virtual Cloud Network OCID is required
+      label: Virtual Cloud Network
+      placeholder: VCN Display Name
       invalid: Virtual Cloud Network OCID must be a well-formed OCID
     controlPlaneSubnet:
-      help: The OCID of an existing subnet to be used by the control plane
-      label: Control Plane Subnet OCID
-      placeholder: Control Plane Subnet OCID
-      required: Control Plane Subnet OCID is required
+      label: Control Plane Subnet
       invalid: Control Plane Subnet OCID must be a well-formed OCID
     loadBalancerSubnet:
-      help: The OCID of an existing subnet to be used for load balancers
-      label: Load Balancer Subnet OCID
-      placeholder: Load Balancer Subnet OCID
-      required: Load Balancer Subnet OCID is required
+      label: Load Balancer Subnet
       invalid: Load Balancer Subnet  OCID must be a well-formed OCID
     workerNodeSubnet:
-      help: The OCID of an existing subnet to be used by the worker nodes
```

Comment:
why do these labels get removed from some elements, but remain on others?

Reply:
doing some cleanup, these I added earlier but ended up not using